packages/esm-reports-app/src/components/reports.resource.tsx

```diff
@@ -222,3 +222,28 @@ function convertParametersToString(data: any): string {
 
   return finalString;
 }
+
+export function getReportDataObservable(
```

Comment:
Generally, we've strongly preferred to use React hooks, `useSWR()`, and `openmrsFetch()` to do data fetching rather than relying on RXJS. It would be preferrable to follow the same pattern here.